test/public/dataPasses/overviewPerLhcPeriod.test.js

```diff
@@ -67,6 +69,41 @@ module.exports = () => {
         };
 
         await validateTableData(page, new Map(Object.entries(tableDataValidators)));
+        const names = await getAllDataFields(page, 'name');
+        const runs = await getAllDataFields(page, 'associatedRuns');
+        const anchoreds = await getAllDataFields(page, 'anchoredSimulationPasses');
+        expect(names.map((name, index) => ({
+            name,
+            runsCount: Number(runs[index].split('\n')[0]) || 0,
+            simulationPassesCount: Number(anchoreds[index].split('\n')[0]) || 0,
+        }))).to.have.all.deep.members([
+            {
+                name: 'LHC22b_apass2',
+                runsCount: 3,
+                simulationPassesCount: 1,
+            },
+            {
+                name: 'LHC22b_apass1',
+                runsCount: 3,
+                simulationPassesCount: 1,
+            },
+        ]);
+    });
+
+    it('can navigate to runs per data pass page', async () => {
+        await goToPage(page, 'data-passes-per-lhc-period-overview', { queryParameters: { lhcPeriodId: 2 } });
+        await waitForNavigation(page, () => pressElement(page, 'tbody tr td:nth-of-type(2)'));
+        const url = new URL(page.url());
+        expect(url.searchParams.get('page')).to.be.equal('runs-per-data-pass');
+        expect(Number(url.searchParams.get('dataPassId'))).to.be.a('Number');
```

Comment:
Why not using `checkMismatchingUrlParam` ?

Reply:
Because: checking correctness of the feature should concluded with checking current url, `mismatching stuff` would be helpful in case of checking exact match, for that I would need to have the `id`, which can me access only from the `model` (bad due to mixing layers). Other option is to fetch directly from endpoint, a bit cumbersome.

Reply:
refactored to expect exact match